src/providers/export/tensorFlowPascalVOC.ts

```diff
@@ -155,16 +197,147 @@ item {
         const annotationsFolderName = `${exportFolderName}/Annotations`;
         await this.storageProvider.createContainer(annotationsFolderName);
 
+        const annotationTemplate = `
+<annotation verified="yes">
```

Comment:
Please move out to external file and see if we can import it like we do with JSON files.